$MirOS: ports/emulators/qemu/snapshot/patches/patch-qemu-doc_texi,v 1.1 2008/05/07 15:53:11 tg Exp $
$OpenBSD: patch-qemu-doc_texi,v 1.3 2008/04/28 22:52:38 todd Exp $
--- qemu-doc.texi.orig	Wed Sep 23 19:01:08 2009
+++ qemu-doc.texi	Sat Oct 17 16:35:43 2009
@@ -1343,6 +1343,16 @@ These variables are not used by Open Hac
 
 @end table
 
+Kernel QEMU options:
+@table @option
+
+@item -kernel-kqemu
+Enable KQEMU full virtualization (default is user mode only).
+
+@item -no-kqemu
+Disable KQEMU kernel module usage.
+@end table
+
 @c man end
 
 
